iterators.tex

```diff
@@ -6394,26 +6397,32 @@
 only if \tcode{size(t)} returns the number of elements in \tcode{t}.}
 
 \pnum
-For any type \tcode{T}, \tcode{is_sized_range<T>} derives from
-\tcode{true_type} if \tcode{T} \oldtxt{models}\newtxt{meets the syntactic requirements of}
-\tcode{\xname{SizedRangeLike}}, and \tcode{false_type} otherwise.
+For a\oldtxt{ny} type \tcode{T} \newtxt{that satisfies \tcode{\xname{SizedRangeLike<T>}}},
+\tcode{is_sized_range<T>} derives from \tcode{true_type} if \oldtxt{\tcode{T} models
+\tcode{\xname{SizedRangeLike}}} \newtxt{\tcode{Same<T, remove\_cv\_t<remove\_reference\_t<T>{}>{}>()} is satisfied}, and \oldtxt{\tcode{false_type}} \newtxt{\tcode{is_sized_range<remove_cv_t<remove_reference_t<T>{}>{}>}} otherwise.
```

Comment:
>> For the specification, we can't refer to meta, though.
>
> There's always see below ;)

One of these days I'll propose Meta for the standard library. It would make so many things so much easier.